functions/src/import-csv.ts

```diff
@@ -76,24 +79,26 @@ async function importCsv(req, res) {
     await Promise.all(inserts);
     const count = inserts.length;
     console.log(`Inserted ${count} rows`);
-    await res.status(HttpStatus.OK).end(JSON.stringify({ count }));
+    res.status(HttpStatus.OK).end(JSON.stringify({ count }));
   });
 
   busboy.on("error", (err) => {
-    console.err("Busboy error", err);
-    next(err);
+    console.error("Busboy error", err);
+    // TODO bring next function back ????
+    // next(err);
   });
-  busboy.end(req.rawBody);
+  // rawBody ???
+  busboy.end(req.body);
```

Comment:
I think we can remove the call to `end` here.

Reply:
Here and every other occasion right?

Reply:
Only here. I think we hand it over to busboy to handle the stream and call the event handlers, which in turn may call end.

Reply:
I meant removing the other `busboy.end(req.body);`, no?

Reply:
Yes, I see there are two instants, one here and one in import-geojson.ts. Those appear to be unnecessary.